### PR TITLE
Liquid API integration

### PR DESCRIPTION
**Pull request by renanmav**
This PR integrates Liquid API into our app.

It adds username claiming through the `getRegistrationOptions` endpoint.

It will follow up with the following API calls `verifyRegistration` and `updateUserAddress`.

Here's a demo of current behavior:

<video src="https://github.com/user-attachments/assets/1d7e6fd1-caf9-4898-bf7e-1f8a62f2f5b4" />

### CODE REVIEW

File: screens/tag/index.tsx [hunk 1]
```diff
@@ -1,48 +1,84 @@
...
+
+      (async () => {
+        if (tag === '' || tag.length < 3) {
+          setFormState('base');
+          return;
+        }
+
+        try {
+          const options = await api.getRegistrationOptions(tag);
+          console.log('registrationOptions', options);
```
Comment (MeisterJustice) [inline screens/tag/index.tsx:56]: we'd have to clean up these logs later

File: store/smartAccount/createSmartAccount.ts [hunk 2]
```diff
@@ -124,18 +96,26 @@ export async function createSmartAccount(username: string): Promise<{
...
+        [
+          {
+            text: 'Try again',
+            onPress: () => {
+              createSmartAccount(username, registrationOptions);
+            },
+          },
+        ]
+      );
+    }
```
Comment (MeisterJustice) [inline store/smartAccount/createSmartAccount.ts:118]: A side comment. This is where the passkey lib ought to call the permission request. Weird they don't

**Review by MeisterJustice — APPROVED** [review]

File: screens/tag/index.tsx (hunk 1 above)
Comment (renanmav) [reply screens/tag/index.tsx:56]: Yes, we'll do later

File: store/smartAccount/createSmartAccount.ts (hunk 2 above)
Comment (renanmav) [reply store/smartAccount/createSmartAccount.ts:118]: I hope that's gonna be resolved by #36

**Pull request merged**